frontend/javascripts/oxalis/view/right-border-tabs/tree_hierarchy_view.tsx

```diff
@@ -94,7 +94,7 @@ function TreeHierarchyView(props: Props) {
       // For some React rendering/timing reasons, the target element might not be rendered yet. That messes with calculating the offsets for scrolling. Hence delay this a bit
       setTimeout(() => {
         if (treeRef.current) treeRef.current.scrollTo({ key: activeTreeKey, align: "auto" });
-      }, 30);
+      }, 50);
```

Comment:
I increased the time a little as it happened to me that under quite some load chrome wasn't fast enough to expand the groups before the scrollTo call was made and thus the scrollTo was ignored. The 50 ms might be more "slow browser tolerant"